docs/keto/guides/userset-rewrites.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Userset Rewrites
+---
+
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import CodeFromRemote from "@theme/CodeFromRemote"
+const sha = "fc39b65f8be9b23bbb4e19480706964d61505c5e"
+const file = (path) => `https://github.com/ory/keto/blob/${sha}/contrib/rewrites-example/${path}`
+
+This guide will explain how to configure your your namespaces and relations using the
+[Ory Permission Language](../reference/ory-permission-language).
+
+The example describes a file store. Individual files are organized in a folder hierarchy, and can be accessed by users or groups
+of users. Through the Ory Permission Language we can specify that if a user has access to a folder, the user also has access to
+all files in that folder.
+
+## Setup and Configuration
+
+First, [install Keto](../install.mdx). Next, you need the following `keto.yaml` (at `contrib/rewrites-example/keto.yaml`):
+
+<CodeFromRemote src={file("keto.yaml")} />
+
+In the _namespaces_ key, you see the line `config: file://./namespaces.keto.ts`. This needs to point to your namespace
+configuration in the Ory Permission Language.
+
+:::info Backwards compatibility
+
+Namespace configurations without the Ory Permission Language are still supported by either specifying the namespaces directly or a
+"naked" URI, e.g. `namespaces: file://.namespaces.yaml`.
+
+:::
+
+Next, we create a file with our namespace configuration as follows:
+
+<CodeFromRemote src={file("namespaces.keto.ts")} />
+
+:::tip Editor support
+
+If you are using a text editor with TypeScript support, you can get extra help when using the Ory Permission Language. Just make
+sure that the file `contrib/rewrites-examples/lib.ts` is in the same folder as the file you are editing. It contains all
+definitions to type-check the config.
+
+In the future, we will make the lib file available at an easy-to-import location.
+
+:::
+
+### Starting Ory Keto
+
+After you created both configuration files (keto.yaml and namespaces.keto.ts), you can start Ory Keto with:
+
+```sh
+$ keto serve --config ./path/to/keto.yaml
+```
+
+## Creating the tuples
+
+Now that Ory Keto is running, we can create the tuples either through the Keto CLI.
```

Comment:
I think you wanted to say that tuples can be created through the CLI or by loading files, right? The sentence looks unfinished the way it is now ;)